mbed_app.json

```diff
@@ -28,7 +28,9 @@
     "macros": ["MBEDTLS_USER_CONFIG_FILE=\"mbedtls_mbed_client_config.h\""],
     "target_overrides": {
         "*": {
-            "target.features_add": ["NANOSTACK", "LOWPAN_ROUTER", "COMMON_PAL"],
+            "target.features_add": ["LWIP", "NANOSTACK", "LOWPAN_ROUTER", "COMMON_PAL"],
+            "lwip.ipv4-enabled":false,
+            "lwip.ipv6-enabled":true,
```

Comment:
This will break IPv4 connectivity probably. I think we should still default to IPv4 and mention in readme the needed switch for IPv6.